### ISSUE TITLE
CATS 0.19 not compatible with Blender 3.2

### ISSUE THREAD

**Issue by crist0393** [absolute-quantum/cats-blender-plugin#495]
CATS is no longer compatible with Blender 3.2, it throws an error when I import an mmd model,

**Comment by treschlet** [comment]
Yeah, same. Looks like Blender 3.2 is using a newer version of python, which no longer allows implicit downconversion from float to int, which seems to break a number of things. Possibly other issues as well.

**Comment by Xiarno** [comment]
Can confirm. Doesn't work anymore.

**Comment by imagitama** [comment]
Have you tried the `development` branch?

**Comment by Kaelygon** [comment]
I tested both 0.19 and development, bot giving ModuleNotFoundError: No module named 'requests'

**Comment by Kaelygon** [comment]
> 
running command
`python -m pip install requests`
seems to have done the trick

**Comment by Spectre-Slayer** [comment]
how  did you do it?

**Comment by pandages** [comment]
I'm having the same issue.  I saw the same float->int mention in a python error log within Blender.

I tried installing standalone python, updating the pip, and installing the module 'requests' but it didn't improve anything for me.

**Comment by davide-dc-dev** [comment]
This addon is compatible only with blender 2.93 and below. My advice is to install blender 2.93 until They decide to support blender 3.x

**Comment by Spokeek** [comment]
You can make this project work with more recent versions of blender by downloading the project as a ZIP from the "development" branch.

Then import is as an Addon and it should work as expected in more recent versions of Blender.

**Comment by imagitama** [comment]
Yep I got it working on 3.x by downloading the development branch as ZIP and adding it as an addon
